native/cocos/renderer/gfx-base/GFXDef-common.h

```diff
@@ -1089,6 +1088,8 @@ struct ALIGNAS(8) TextureViewInfo {
     uint32_t levelCount{1};
     uint32_t baseLayer{0};
     uint32_t layerCount{1};
+    uint32_t basePlane{0};
+    uint32_t planeCount{1};
```

Comment:
It may miss the range of subranges used, such as the texture barrier. However, it may break compatibility.